Source/Scene/CreditDisplay.js

```diff
@@ -385,15 +385,21 @@ define([
         Check.defined('credit', credit);
         //>>includeEnd('debug');
 
+        var imageCredits = this._currentFrameCredits.imageCredits;
+
         if (credit.text === 'Cesium ion') {
             // We don't want to clutter the screen with the Cesium logo and the Cesium ion
             // logo at the same time. Since the ion logo is required, we just replace the
-            // Cesium logo.  This will also add the logo if the Cesium one was removed.
-            this._currentFrameCredits.imageCredits[cesiumCredit.id] = credit;
+            // Cesium logo or add the logo if the Cesium one was removed.
+            if (defined(imageCredits[cesiumCredit.id])) {
```

Comment:
Should this be `CreditDisplay.cesiumCredit`?

Reply:
No, this is specifically checking if the `cesiumCredit` we created at load time is in the scene, if someone assigned `CreditDisplay.cesiumCredit`, we want this if check to fail so that we add the ionCredit instead of replacing whatever `CreditDisplay.cesiumCredit` is.

Reply:
👍